examples/00-mapdl-examples/composite_dcb.py

```diff
@@ -304,13 +304,20 @@
 # Use PyDPF to visualize the crack opening throughout the simulation as
 # an animation.
 
-# Upload file to DPF server
-temp_directory = tempfile.gettempdir()
-rst_path = mapdl.download_result(temp_directory)
-server_file_path = dpf.upload_file_in_tmp_folder(rst_path)
+try:
+    # ONLY IF DPF SERVER DEPLOYED WITH gRPC COMMUNICATION
+    # Upload file to DPF server
+    temp_directory = tempfile.gettempdir()
+    rst_path = mapdl.download_result(temp_directory)
+    dpf.connect_to_server()
+    server_file_path = dpf.upload_file_in_tmp_folder(rst_path)
+    data_src = dpf.DataSources(server_file_path)
+except:
```

Comment:
Next time, try to not use a bare ``except``. It is a bad practice and it is flagged by our code analysis workflow.